go/kbfs/libkbfs/keybase_daemon_rpc.go

```diff
@@ -465,6 +465,6 @@ func newNotifyServiceHandler(config Config, log logger.Logger) keybase1.NotifySe
 func (k *KeybaseDaemonRPC) FavoritesChanged(ctx context.Context,
 	uid keybase1.UID) error {
 	k.log.Debug("Received FavoritesChanged RPC.")
-	k.config.KBFSOps().RefreshCachedFavorites(ctx)
+	go k.config.KBFSOps().RefreshCachedFavorites(ctx, true)
```

Comment:
And if you do use a `go` here, you have to make a new context, since `ctx` will probably be canceled by the RPC layer as soon as the function returns.